app/k1util/k1util.go

```diff
@@ -0,0 +1,173 @@
+// Copyright © 2022 Obol Labs Inc.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the Free
+// Software Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of  MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+// more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program.  If not, see <http://www.gnu.org/licenses/>.
+
+// Package k1util provides helper function for working with secp256k1 keys.
+package k1util
+
+import (
+	"encoding/hex"
+	"os"
+
+	k1 "github.com/decred/dcrd/dcrec/secp256k1/v4"
```

Comment:
suggest that we add a lint for this alias to always require it.

Reply:
I have created a ticket for this - have a look: https://github.com/ObolNetwork/charon/issues/1684